apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.Snippets/UrlSignerSnippets.cs

```diff
@@ -119,5 +124,88 @@ public UrlSignerSnippets(StorageSnippetFixture fixture)
 
             await client.DeleteObjectAsync(bucketName, destination);
         }
+
+        // Sample: IamServiceBlobSigner
+        internal sealed class IamServiceBlobSigner : UrlSigner.IBlobSigner
+        {
+            private readonly IamService _iamService;
+            public string Id { get; }
+
+            internal IamServiceBlobSigner(IamService service, string id)
+            {
+                _iamService = service;
+                Id = id;
+            }
+
+            public string CreateSignature(byte[] data) =>
+                CreateRequest(data).Execute().Signature;
+
+            public async Task<string> CreateSignatureAsync(byte[] data, CancellationToken cancellationToken)
+            {
+                var request = CreateRequest(data);
```

Comment:
Should the snippets be using `var`?

Reply:
Ooh, probably not. I'm not *as* bothered for this snippet as others (as it's an example implementation that can be copy/pasted) but I'll fix it anyway.